start-site/src/main/resources/application.yml

```diff
@@ -1283,17 +1285,6 @@ initializr:
               description: Using Key Vault
             - rel: reference
               href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-spring-boot-starters/azure-keyvault-secrets-spring-boot-starter
-        - name: Azure Storage
```

Comment:
Can you share some  context as why this is removed?

Reply:
Hi, @snicoll :

* This starter (azure-storage-spring-boot-starter) is deprecated in 2.3.0, we will add another starter (spring-starter-azure-storage) for storage soon
* The azure-storage-spring-boot-starter needs to be available for 2.2.x, so we cannot delete it completely.

For the 2 points, should I delete the content in this PR?

Reply:
Thanks for sharing the context. I can take care of that as part of merging.

Reply:
Thank you.